src/main/java/com/programmers/springbootbasic/VoucherApplication.java

```diff
@@ -0,0 +1,25 @@
+package com.programmers.springbootbasic;
+
+import com.programmers.springbootbasic.exception.AppExceptionHandler;
+import com.programmers.springbootbasic.infrastructure.IO.Console;
+import com.programmers.springbootbasic.mediator.RequestProcessor;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+@SpringBootApplication
+public class VoucherApplication {
+
+    public static void main(String[] args) {
+        ApplicationContext applicationContext = SpringApplication.run(VoucherApplication.class,
+            args);
+
+        AppExceptionHandler exceptionHandler = applicationContext.getBean(
+            AppExceptionHandler.class);
+
+        while (true) {
+            exceptionHandler.handle();
+        }
+    }
```

Comment:
메서드로 추출하여 가독성을 높이는 것도 괜찮을 거 같습니다.

예를 들면 다음과 같이..?
```suggestion
handleException(applicationContext);
```

Reply:
[dd3bebd](https://github.com/prgrms-be-devcourse/springboot-basic/pull/902/commits/dd3bebd82c92776b195c58bd03a2ad72b35eb8ae)

분리해 보았습니다! 현재는 메인 메서드 밑에 static 으로 그냥 정의 해 뒀는데, 혹시 따로 class 를 빼는 것이 적절할까요? 
제 생각엔 
1. 컨텍스트에서 빈을 가져오고
2. 해당 빈을 while 로 실행 시키며
3. 종료 상태가 된다면 감지해서 while 문을 탈출해 종료하는
로직이 어플리케이션 실행을 도와주는 역할 같아서 AppRunner 같은걸로 빼도 어색하지 않을 것 같아서 여쭤봅니다!

Reply:
클래스로 따로 빼서 책임을 나누려는 의도는 좋은 것 같습니다

단, 클래스로 나눴을때 어떠한 이점을 가질 수 있는지 생각해보면 좋을 것 같습니다.

다음과 같은 부분을 고려해보시는 건 어떨까요?

1. 해당 로직이 실행 이외에도 여러가지 많은 기능을 가지고 있는지
2. 추후에 해당 로직이 확장될 가능성이 있는지
3. 다른 곳에서 해당 로직을 재사용할 가능성이 있는지

이 부분에 대해 고민하고 클래스를 나누는게 과도하다는 생각이 들면 유지하는게 좋을 것 같습니다,,!

Reply:
추후에 앱 실행 전 초기화 시키는 일 정도 하는 것으로 확장 될 수도 있다는 생각이 드는데... 또 초기화는 초기화 class 에서 따로 하는 것이 맞다는 생각이 들어! 과하다는 판단이 들어 분리하지 않겠습니다!!! 판단 기준을 적어주신 점 너무 명쾌하네요...! 해당 부분은 코드리뷰 때 팀원 분들과도 나누겠습니다🙏